### PR TITLE
[SMD-323] validation error updates

### PR DESCRIPTION
**Pull request by bobbi-hbrown**
https://dluhcdigital.atlassian.net/jira/software/c/projects/SMD/boards/140?quickFilter=815&selectedIssue=SMD-323

Moving the errors away from const.py and adding the updated errors into a similar structure in messages.py. 

### Change description
_A brief description of the pull request_

- [X] Unit tests and other appropriate tests added or updated
- [X] README and other documentation has been updated / added (if needed)
- [X] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
_If manual testing is needed, give suggested testing steps_

### CODE REVIEW

**Review by cyrusdobbs — CHANGES_REQUESTED** [review]
Looks good just a small change

**Review by gidsg — APPROVED** [review]

**Review by cyrusdobbs — APPROVED** [review]
LGTM

File: core/validation/messages.py [hunk 1]
```diff
@@ -39,3 +41,18 @@
...
+    "Fund Type": "Cell E7 in the “project admin” must contain a fund type from the dropdown list provided."
+    " Do not enter your own content.",
+    "Place Name": "Cell E8 in the “project admin” must contain a place name from the dropdown list provided. "
+    "Do not enter your own content.",
+    "Reporting Period": "Cell B6 in the “start here” tab must say “1 April 2023 to 30 September 2023”. Select this "
+    "option from the dropdown list provided.",
+    "Not Excel File": "The selected file must be an Excel file",
+    "Wrong Template": "The selected file must be the Town Deals and Future High Streets Fund Reporting Template"
+    f" ({TF_ROUND_4_TEMPLATE_VERSION})",
+    "Form Version": "You have submitted the wrong reporting template. Make sure you submit Town Deals and Future High "
```
Comment (Connor-OS) [inline core/validation/messages.py:56]: The "You have submitted the wrong reporting template" pat of this message isn't in the current version from design unless I'm mistaken. I'm looking at SMD-233 and it seems to be removed

**Review by Connor-OS — CHANGES_REQUESTED** [review]
I think one of the messages is still in the old format, can you check that one against what design gave us

File: core/validation/messages.py (hunk 1 above)
Comment (cyrusdobbs) [reply core/validation/messages.py:56]: good spot looks like it should be "The selected file must be the Town Deals and Future High Streets Fund Reporting Template ({VERSION})"

File: core/validation/messages.py [hunk 2]
```diff
@@ -40,3 +42,18 @@
...
+    " Do not enter your own content.",
+    "Place Name": "Cell E8 in the “project admin” must contain a place name from the dropdown list provided. "
+    "Do not enter your own content.",
+    "Reporting Period": "Cell B6 in the “start here” tab must say “1 April 2023 to 30 September 2023”. Select this "
+    "option from the dropdown list provided.",
+    "Not Excel File": "The selected file must be an Excel file",
+    "Wrong Template": "The selected file must be the Town Deals and Future High Streets Fund Reporting Template"
+    f" ({TF_ROUND_4_TEMPLATE_VERSION})",
+    "Form Version": "The selected file must be the Town Deals and Future High Streets Fund Reporting Template "
+    f"({TF_ROUND_4_TEMPLATE_VERSION})",
```
Comment (cyrusdobbs) [inline core/validation/messages.py:58]: One of these must be redundant I assume?

File: core/validation/messages.py [hunk 3]
```diff
@@ -40,3 +42,18 @@
...
+
+# PRE-VALIDATION
+PRE_VALIDATION_MESSAGES = {
+    "Fund Type": "Cell E7 in the “project admin” must contain a fund type from the dropdown list provided."
+    " Do not enter your own content.",
+    "Place Name": "Cell E8 in the “project admin” must contain a place name from the dropdown list provided. "
+    "Do not enter your own content.",
+    "Reporting Period": "Cell B6 in the “start here” tab must say “1 April 2023 to 30 September 2023”. Select this "
+    "option from the dropdown list provided.",
+    "Not Excel File": "The selected file must be an Excel file",
```
Comment (cyrusdobbs) [inline core/validation/messages.py:54]: This one can be removed

**Review by Connor-OS — APPROVED** [review]

**Pull request merged**